### PR TITLE
Add comment on issues marked 'good first issue'

### PR DESCRIPTION
**Pull request by oscargus**
## PR Summary

Who knows if this works? @melissawm feel free to update the text.

## PR Checklist

**Documentation and Tests**
- [ ] Has pytest style unit tests (and `pytest` passes)
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] New plotting related features are documented with examples.

**Release Notes**
- [ ] New features are marked with a `.. versionadded::` directive in the docstring and documented in `doc/users/next_whats_new/`
- [ ] API changes are marked with a `.. versionchanged::` directive in the docstring and documented in `doc/api/next_api_changes/`
- [ ] Release notes conform with instructions in  `next_whats_new/README.rst` or `next_api_changes/README.rst`

### CODE REVIEW

**Comment by oscargus** [comment]
@melissawm friendly reminder to have a look and possibly suggest a better wording. (I think we will have to try it to see how it works, but this is a reasonably well used action.)

File: .github/workflows/good-first-issue.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      issues: write
+    steps:
+      - name: Add comment
+        uses: peter-evans/create-or-update-comment@v3
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Please see https://matplotlib.org/stable/devel/index for more information if needed.
+            We do not assign issues, but unless there is a PR for this issue already, feel free
+            to just start working on it. Do not hesitate to ask if something needs clarification.
```
Comment (melissawm) [inline .github/workflows/good-first-issue.yml:20]:
```suggestion
            Please see [our Contributing guide](https://matplotlib.org/stable/devel/index) for
            more information on how to contribute.
            **We do not assign issues**, and unless there is a PR for this issue already, feel free
            to start working on it. Do not hesitate to ask if something needs clarification - see
            also [our different communication channels](https://matplotlib.org/devdocs/devel/contributing.html#get-connected). 
```

**Review by melissawm — COMMENTED** [review]
Sorry I missed this! Here's a suggestion for the wording - I also added a direct link to the "get connected" section.

**Review by melissawm — COMMENTED** [review]
Sorry I missed this! Here's a suggestion for the wording - I also added a direct link to the "get connected" section.

**Comment by QuLogic** [comment]
pre-commit doesn't like it; some trailing whitespace slipped in.

File: .github/workflows/good-first-issue.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      - name: Add comment
+        uses: peter-evans/create-or-update-comment@v3
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Please see [our Contributing guide](https://matplotlib.org/stable/devel/index) for
+            more information on how to contribute.
+            **We do not assign issues**, and unless there is a PR for this issue already, feel free
+            to start working on it. Do not hesitate to ask if something needs clarification - see
+            also [our different communication channels](https://matplotlib.org/stable/devel/contributing.html#get-connected).
```
Comment (oscargus) [inline .github/workflows/good-first-issue.yml:22]:
```suggestion
            Please see [our Contributing guide](https://matplotlib.org/stable/devel/index) for
            more information on how to contribute.
            **We do not assign issues**, and unless there is a PR for this issue already, feel free
            to start working on it. Do not hesitate to ask if something needs clarification - see
            also [our different communication channels](https://matplotlib.org/stable/devel/contributing.html#get-connected).
```

**Comment by oscargus** [comment]
Fixed that. Make sure to squeeze when merging.

**Review by QuLogic — APPROVED** [review]

**Comment by oscargus** [comment]
It seems to work: #25942 (although the link text makes the text in the comment line break in an ugly way because of the line length limitation).

**Pull request merged**